test/unit/kubernetes-deploy/kube_config_test.rb

```diff
@@ -4,48 +4,13 @@
 class KubeConfigTest < KubernetesDeploy::TestCase
   def setup
     WebMock.disable_net_connect!
-    set_google_env_vars
   end
 
   def teardown
     WebMock.allow_net_connect!
   end
 
-  def test_auth_use_default_gcp_success
```

Comment:
Is this test invalid now, or are you removing it because we no longer own the code that makes it work?

Reply:
I removed it because it’s no longer our responsibility, the same way we don’t have test coverage for OIDC for example.